lib/puppet/network/formats.rb

```diff
@@ -151,7 +151,7 @@ def render(datum)
     end
 
     # ...or pretty-print the inspect outcome.
-    return json.render(datum)
+    return JSON.pretty_generate(datum)
```

Comment:
This is causing a problem in CI when `datum` is `true`, because our `json.render` method accepts datum that JSON.pretty_generate does not:

```
(rdb:1) datum
true
(rdb:1) datum.class
TrueClass
(rdb:1) json.render(datum)
"true"
(rdb:1) JSON.pretty_generate(datum)
*** JSON::GeneratorError Exception: only generation of JSON objects or arrays allowed
```